src/scraper/sites/anime/animelatinohd/AnimeLatinoHD.ts

```diff
@@ -2,104 +2,127 @@ import * as cheerio from "cheerio";
-                    if ($(e).text() == sel_lang[0].type) {
-                        f_index = Number($(e).val())
-                    }
-                })
-            } else {
-                $("#languaje option").each((_i, e) => {
-                    f_index = Number($(e).val())
-                })
-            }
-
-            await Promise.all(animeEpisodeParseObj.players[f_index].map(async (e: { server: { title: string; }; id: string; }) => {
-               //const min = await axios.get("https://filemoon.sx/e/smone1s7jjxv/CYM01HNMCGTSKT")
-               //const pageload = await BrowserHandler("https://animelatinohd.com/")
-               
-                const Server: EpisodeServer = {
-                    name: e.server.title,
-                    url: "",
-                }
-                //const cookies = [{name: 'v_id', value: "https://api.animelatinohd.com/stream/"+e.id},];
-                Server.url = "https://api.animelatinohd.com/stream/" + e.id
-                Server.name = e.server.title
-                
-               
-                
-                //await pageload.page.setCookie(...cookies)
-                /*await pageload.page.evaluate(()=>{
+  }
+  async GetEpisodeServers(episode: string, lang: string): Promise<Episode> {
+    try {
+      const number = episode.substring(episode.lastIndexOf("-") + 1);
+      const anime = episode.substring(0, episode.lastIndexOf("-"));
+      const langType = [
+        { lang: "es", type: "Latino" },
+        { lang: "jp", type: "Subtitulado" },
+      ];
+
+      const { data } = await axios.get(`${this.url}/ver/${anime}/${number}`);
+      const $ = cheerio.load(data);
+
+      const animeEpisodeParseObj = JSON.parse($("#__NEXT_DATA__").html()).props
+        .pageProps.data;
+
+      const AnimeEpisodeInfo: Episode = {
+        name: animeEpisodeParseObj.anime.name,
+        url: `/anime/animelatinohd/episode/${episode}`,
+        number: number,
+        image: "",
+        servers: [],
+      };
+
+      const sel_lang = langType.filter((e) => e.lang == lang);
+      let f_index = 0;
+
+      if (sel_lang.length) {
+        $("#languaje option").each((_i, e) => {
+          if ($(e).text() == sel_lang[0].type) {
+            f_index = Number($(e).val());
+          }
+        });
+      } else {
+        $("#languaje option").each((_i, e) => {
+          f_index = Number($(e).val());
+        });
+      }
+
+      await Promise.all(
+        animeEpisodeParseObj.players[f_index].map(
+          async (e: { server: { title: string }; id: string }) => {
+            //const min = await axios.get("https://filemoon.sx/e/smone1s7jjxv/CYM01HNMCGTSKT")
+            //const pageload = await BrowserHandler("https://animelatinohd.com/")
+
+            const Server: EpisodeServer = {
+              name: e.server.title,
+              url: "",
+            };
+            //const cookies = [{name: 'v_id', value: "https://api.animelatinohd.com/stream/"+e.id},];
```

Comment:
Please, don't put commented code, delete or do whatever you want with that code.